public/locales/en-US/page-meta-chain.json

```diff
@@ -67,5 +67,27 @@
     "social": {
       "title": "Get updates through social channels"
     }
+  },
+  "faq": {
+    "title": "Frequently Asked Questions",
+    "items": [
+      {
+        "title": "What is Meta Chain?",
+        "desc": "Meta Chain is a Layer 2 blockchain that allows users to build scalable dApps on Bitcoin and Ethereum."
+      },
+      {
+        "title": "How is Meta Chain related to DeFiChain?",
+        "desc": "MetaChain expands the scope for DeFiChain, allowing it to link to various chains by enhancing interoperability."
+      },
+      {
+        "title": "How can I build on Meta Chain?",
+        "desc": "Refer to the documentation here for more details.",
```

Comment:
checking `here` in the UI is quite prone to error, it works but i prefer we do it in a more future proof way.
`desc` can be an array of object with type `{value: string; type: 'string' | 'link'; link?: string}`,